<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Zea mays</em></h2>
<p><em>Zea mays</em> (maize) has the highest world-wide production of all grain crops, yielding 875 million tonnes in 2012 (<a href="http://faostat.fao.org/">http://faostat.fao.org/</a>). Although a food staple in many regions of the world, most is used for animal feed and ethanol fuel. Maize was domesticated from wild teosinte in Central America and its cultivation spread throughout the Americas by Pre-Columbian civilizations. In addition to its economic value, maize is an important model organism for studies in plant genetics, physiology, and development. It has a large genome of of about 2.4 gigabases with a haploid chromosome number of 10 (Schnable <em>et al</em>, 2009; Zhang <em>et al</em>, 2009). Maize is distinguished from other grasses in that its genome arose from an ancient tetraploidy event unique to its lineage.</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>
<p>The complete genome sequence of <em>Zea mays</em> cv. B73 (RefGen_v1) was published in 2009 by the NSF-funded Maize Genome Sequencing Project (Schnable <em>et al</em>, 2009). The high-quality assembly was accomplished by a strategy of sequencing individual BAC clones along a minimum tiling path anchored to genetic and physical maps (Schnable <em>et al</em>, 2009; Wei, Zhang <em>et al</em>, 2009]. This version of the assembly (RefGen_v3) incorporates additional contigs assembled from whole genome shotgun sequencing reads (Olson <em>et al</em>, manuscript in preparation). These contigs were selected because they include portions of full length cDNAs that were not covered by the BAC based assembly. The contigs were inserted into gaps based on a synteny-refined genetic map. This genetic map was also used to rearrange some clones.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>
<p>Genes were originally annotated using both an evidence-based approach (e.g. using cDNA and EST data) and an <em>ab initio</em> approach (FGENESH), which were combined to give a unique non-overlapping gene set. New and updated gene models are limited to the regions where new contigs were inserted.</p>
<p><strong>MAKER-P Gene Models<br /></strong></p>
<p>In addition to the familiar "Gramene Gene" annotation track, the maize browser now includes a new "MAKER-P_genes" track, providing gene models annotated using<!--nbsp--> <!--nbsp--> MAKER-P software (Law <em>et al</em>, 2015).<!--nbsp--> <!--nbsp--> Incorporating evidence from a large number of RNA-seq studies, this new set includes 4,466 additional protein-coding genes not present in the 5b+ annotation build, and improves the annotation of UTR's in 1,393 gene models.<!--nbsp--> <!--nbsp--> This set excludes 2,647 5b+ gene models that lack support in the MAKER-P build (Law <em>et al</em>, 2015).</p>
<p><strong><em>Zea mays</em> nascent transcriptomes<br /></strong></p>
<p>Nascent transcriptomes of wild-type (WT) and RNA Pol D1 (<em>rpd1</em>) mutant seedlings are available as RNA aligments in the genome browser. The source of these data is the publication by Erhard <em>et al</em> (2015).</p>
<p><strong>Long non-coding RNAs</strong></p>
<p>The results of a genomewide screen for long non-coding RNAs (lncRNAs) are shown as RNA alignments of 20,163 putative lncRNAs, including ~1,704 high-confidence lncRNAs, as published by Li <em>et al<!--nbsp--> <!--nbsp--></em>(2014).</p>
<!-- {annotation} -->
<!-- {regulation} --><a name="regulation"></a>
<h3 id="regulation">Regulation</h3>
<p><strong>Gene expression probes<br /></strong></p>
<p>Oligo probes from the <a href="http://www.affymetrix.com/estore/browse/products.jsp?productId=131468">GeneChip Maize Genome Array</a> have been aligned using the standard <a href="http://plants.ensembl.org/info/docs/microarray_probe_set_mapping.html">Ensembl 2-step mapping procedure</a>. For example, see the the results for <a href="http://goo.gl/o3vfL">Zm.155.1.A1_a_at</a>.</p>
<p><strong>DNA methylation<br /></strong></p>
<p>Genomewide patterns of DNA methylation for two maize inbred lines, B73 and Mo17, are now diplayed on the maize genome browser. Cytosine methylation in symmetric (CG and CHG, where H is A, C, or T) context is associated with DNA replication and histone modification. CG (65%) and CHG (50%) methylation is also highest in transposons. Source: Maize methylome publication by Regulski <em>et al<!--nbsp--> <!--nbsp--></em>(2013).</p>
<!-- {regulation} -->
<!-- {variation} --><a name="variation"></a>
<h3 id="variation">Variation</h3>
<h4>HapMap2 dataset</h4>
<p>A variation set which comprises the maize HapMap2 data (Chia <em>et al</em>, 2012). This dataset incorporates 55 million SNPs and indels identified in a collection of 103 pre-domesticated and domesticated <em>Zea mays</em> varieties, including a representative from the sister genus, <em>Tripsacum dactyloides</em> (Eastern gamagrass). Each line was sequenced to an average of 4.5X coverage using the Illumina GAIIx platform. The reads can be accessed from the SRA, with accession ID: SRA051245. Reads were mapped to the B73 reference genome using a combination of Bowtie, Novoalign and SOAP. The variations were scored by taking into account identity-by-descent blocks that are shared among the lines.</p>
<h4>The <a href="http://www.panzea.org/lit/data_sets.html">Panzea 2.7</a> genotyped-by-sequencing (GBS) dataset</h4>
<p>This variation data set consists of 719,472 SNPs (excluding 332 SNPs that were removed for mapping to scaffolds) typed in 16,718 maize and teosinte lines, and grouped in 14 overlapping populations according to the germplasm set in the corresponding <a href="http://www.panzea.org/dynamic/derivative_data/genotypes/AllZeaGBSv2.7_publicSamples_metadata20140411.xlsx">metadata table</a>.</p>
<!-- {variation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Links</h3>
<ul><li><a title="MaizeSequence.org" href="ftp://ftp.gramene.org/pub/gramene/maizesequence.org/">MaizeSequence.org FTP site</a></li>
<li><a title="MaizeGDB" href="http://www.maizegdb.org/">MaizeGDB - Look here for RefGen_v2 on the generic genome browser<br /></a></li>
</ul>
<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/19936050">A genome-wide characterization of microRNA genes in maize</a>.<br>Zhang L, Chia JM, Kumari S, Stein JC, Liu Z, Narechania A, Maher CA, Guill K, McMullen MD, Ware D. 2009. PLoS Genet.. 5:e1000716.</li>
<li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/22660545">Maize HapMap2 identifies extant variation from a genome in flux</a>.<br>Chia JM, Song C, Bradbury PJ, Costich D, de Leon N, Doebley J, Elshire RJ, Gaut B, Geller L, Glaubitz JC et al. 2012. Nat. Genet.. 44:803-807.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/19965430">The B73 maize genome: complexity, diversity, and dynamics</a>.<br>Schnable PS, Ware D, et al.. 2009. Science. 326:1112-1115.</li>
<li><a id="ref-04"></a><a href="http://faostat.fao.org/">FAOSTAT</a>.<br></li>
<li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/22660546">Comparative population genomics of maize domestication and improvement</a>.<br>Hufford MB, Xu X, van Heerwaarden J, Pyhjrvi T, Chia JM, Cartwright RA, Elshire RJ, Glaubitz JC, Guill KE, Kaeppler SM et al. 2012. Nat. Genet.. 44:808-811.</li>
<li><a id="ref-06"></a><a href="http://europepmc.org/abstract/MED/19965431">A first-generation haplotype map of maize</a>.<br>Gore MA, Chia JM, Elshire RJ, Sun Q, Ersoz ES, Hurwitz BL, Peiffer JA, McMullen MD, Grills GS, Ross-Ibarra J et al. 2009. Science. 326:1115-1117.</li>
<li><a id="ref-07"></a><a href="http://europepmc.org/abstract/MED/19936048">Detailed analysis of a contiguous 22-Mb region of the maize genome</a>.<br>Wei F, Stein JC, Liang C, Zhang J, Fulton RS, Baucom RS, De Paoli E, Zhou S, Yang L, Han Y et al. 2009. PLoS Genet.. 5:e1000728.</li>
<li><a id="ref-08"></a><a href="http://europepmc.org/abstract/MED/19936061">The physical and genetic framework of the maize B73 genome</a>.<br>Wei F, Zhang J, Zhou S, He R, Schaeffer M, Collura K, Kudrna D, Faga BP, Wissotski M, Golser W et al. 2009. PLoS Genet.. 5:e1000715.</li>
<li><a id="ref-09"></a><a href="http://europepmc.org/abstract/MED/19936062">A single molecule scaffold for the maize genome</a>.<br>Zhou S, Wei F, Nguyen J, Bechner M, Potamousis K, Goldstein S, Pape L, Mehan MR, Churas C, Pasternak S et al. 2009. PLoS Genet.. 5:e1000711.</li>
<li><a id="ref-10"></a><a href="http://europepmc.org/abstract/MED/19541913">Evidence-based gene predictions in plant genomes</a>.<br>Liang C, Mao L, Ware D, Stein L. 2009. Genome Res.. 19:1912-1923.</li></ol><p>Picture credit: Nicolle Rager Fuller, National Science Foundation.</p>