<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p><img alt="IWGSC" src="/img/IWGSC.jpg" style="float:left; height:100px; padding-right:10px; width:174px" title="IWGSC" />The bread wheat genome in Ensembl Plants is the Chromosome Survey Sequence (CSS) for <a href="/triticum_aestivum" title="Triticum aestivum in Ensembl Plants">Triticum aestivum</a> cv. Chinese Spring, combined with the reference sequence of chromosome 3B, both generated by the <a href="http://www.wheatgenome.org/">International Wheat Genome Sequencing Consortium</a>. The CSS assemblies have been further refined into chromosomal pseudomolecules using POPSEQ data generated by <a href="http://genomebiology.com/2015/16/1/26">Chapman <em>et al</em></a>.<!--nbsp--> <!--nbsp-->The gene models for 3B were annotated by the <a href="http://www6.clermont.inra.fr/umr1095_eng/">GDEC group</a> at INRA. For all other chromosomes, gene models were generated by <a href="http://pgsb.helmholtz-muenchen.de/plant/wheat//iwgsc/index.jsp" title="PGSB">PGSB</a> (<a href="ftp://ftpmips.helmholtz-muenchen.de/plants/wheat/IWGSC/ta_IWGSC_CHANGELOG_2014Jul18.txt">version 2.2</a>). The data presented here has been prepared as part of the <a href="http://www.wheatgenome.org.uk/People.html">Triticeae Genomics for Sustainable Agriculture</a> project, funded by the U.K. <a href="http://www.bbsrc.ac.uk">Biotechnology and Biological Sciences Reseach Council</a>.</p>

<p>See also the wheat homepage at <a href="http://wheat-urgi.versailles.inra.fr/"><img alt="URGI logo" src="/img/logo_URGI.png" style="height:30px; vertical-align:middle" /></a></p>
</div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<p><em>Triticum aestivum</em> (bread wheat) is a major global cereal grain essential to human nutrition. Wheat was one of the first cereals to be domesticated, originating in the <a href="http://en.wikipedia.org/wiki/Fertile_Crescent">fertile crescent</a> around 7000 years ago. Bread wheat is hexaploid, with a genome size estimated at ~17 Gbp, composed of three closely-related and independently maintained genomes that are the result of a series of naturally occurring hybridization events. The ancestral progenitor genomes are considered to be <a href="/Triticum_urartu" title="Triticum urartu in Ensembl Plants"><em>Triticum urartu</em></a> (the A-genome donor) and an unknown grass thought to be related to <em>Aegilops speltoides</em> (the B-genome donor). This first hybridization event produced tetraploid emmer wheat (AABB, <em>T. dicoccoides</em>) which hybridized again with <em><a href="/Aegilops_tauschii" title="Aegilops tauschii in Ensembl Plants">Aegilops tauschii</a></em> (the D-genome donor) to produce modern bread wheat.</p>

<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<p><strong>CS42 TGAC v1 Assembly </strong></p>

<p>250 bp paired-end reads were generated from two CS42 libraries constructed using a PCR-free protocol.<!--nbsp--> <!--nbsp--> In total, 1.1 billion PE reads were generated providing 32.78x coverage of the CS42 genome.<!--nbsp--> <!--nbsp--> The w2rap-contigger (based on DISCOVAR de novo [1]) was used to assemble contigs.<!--nbsp--> <!--nbsp--> The contigger is available in Github [2] and is fully described elsewhere [3].<!--nbsp--> <!--nbsp--> It utilises PCR-free libraries to reduce coverage bias, uses long 250bp reads generated by the latest Illumina sequencing technology and retains the majority of the variation present in the reads when generating contigs.<!--nbsp--> <!--nbsp--> Multiple Nextera long mate-pair libraries were generated for scaffolding with insert sizes ranging from 2-12 Kb.<!--nbsp--> <!--nbsp--> LMP reads were processed using Nextclip [4] and contigs were scaffolded using the SOAPdenovo2 [5] prepare-> map-> scaffold pipeline.<!--nbsp--> <!--nbsp--></p>

<p>Scaffolds were classified into chromosome-arm bins using arm-specific Chromosome Survey Sequence (CSS) reads [6]. Scaffolds from 3B were not separated into short/long arm bins as individual arm datasets were not generated for this chromosome in the CSS project. The ‘sect’ method of KAT [7] was used to compute kmer coverage over each scaffold using each CSS read set. Each non-repetitive kmer in a scaffold was scored proportionally to coverage on each CSS arm and scaffolds were classified using the following set of rules:</p>

<ol><li>Scaffolds with less than 10% of the kmers producing a vote were left as <em>unclassified </em>(marked as Chromosome arm “U”). These are mostly small and/or repetitive sequences.  </li>
	<li>Scaffolds with a top score towards a CSS set <em>at least double </em>the second top score were classified to the highest scoring chromosome arm.  </li>
	<li>Scaffolds with a top score towards a CSS set <em>less than double </em>the second top score were left as <em>unclassified </em>(marked as Chromosome arm “U”, but with the two top scores and CSS sets included in the sequence name). This category contains scaffolds that are classified as combinations of the two arms from the same chromosome, probably due to imprecise identification during flow-sorting. It also contains scaffolds from regions of the genome with specific flow-sorting biases, and assembly chimeras, which will all be investigated further.  </li>
</ol><p>Rather than using a simple length cutoff to include scaffolds in the final assembly, a content filter was applied to the scaffolds classified into each chromosome-arm bin in order to ensure short scaffolds containing unique content were not excluded from the assembly. Scaffolds were sorted by length, longest first. Scaffolds longer than 5Kbp were automatically added to the assembly. Scaffolds between 5Kbp and 500bp were added from longest to smallest if 20% of the kmers in the scaffold were not already present in the assembly. Scaffolds shorter than 500bp were excluded.</p>

<p>For assigned scaffolds, the arm assignment is included in the FASTA identifier. For unassigned scaffolds with more than 10% voting kmers, the highest and second highest vote is included in the FASTA identifier to indicate possible arms.</p>

<p><!--nbsp--> <!--nbsp--></p>

<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>

<h4>IWGSC gene predictions on the Chromosome Survey Sequence (CSS), PGSB/MIPS version 2.2 [1]</h4>

<p>Gene models were derived from the spliced-alignment of publicly available wheat fl-cDNAs and the protein sequences of related grass species, barley, <em>Brachypodium</em>, rice and <em>Sorghum</em>. A large RNA-Seq dataset, covering different tissues and different developmental stages, was used to identify wheat specific genes and additional splice variants. Redundant transcript structures from these different sources were merged.</p>

<p>A total of 99,386 protein-coding genes were predicted, with 193,667 transcripts and splice variants. To simplify display in the genome browser, splice variants are shown on a separate track, which is off by default.</p>

<h4>IWGSC gene predictions on chromosome 3B, <a href="http://www6.clermont.inra.fr/umr1095_eng/">GDEC</a>/INRA version 1.0 [3]</h4>

<p>A total of 5,326 protein-coding genes, 1,938 pseudogenes, and 85% of transposable elements were generated on the 3B chromosome by the GDEC group at INRA. An additional 251 gene models and 188 pseudogenes were annotated on un-anchored 3B scaffolds.</p>

<p>The GDEC gene set replaces the PGSB/MIPS gene set for the purpose of functional annotation and comparative genomics within Ensembl Plants. However, the PGSB/MIPS genes have been projected from the CSS onto the chromosome 3B assembly, and are shown on a separate track.</p>

<h4>Chloroplast and mitochondrial genes</h4>

<p>The chloroplast and mitochondrial gene annotation were imported from their respective<!--nbsp--> <!--nbsp-->ENA entries,<!--nbsp--> <!--nbsp--><a href="http://www.ebi.ac.uk/ena/data/view/KC912694">KC912694</a><!--nbsp--> <!--nbsp-->and<!--nbsp--> <!--nbsp--><a href="http://www.ebi.ac.uk/ena/data/view/AP008982">AP008982</a>.</p>

<h4>Triticeae-CAP predicted transcripts set - Krasileva <em>et al.</em> [4]</h4>

<p>Predicted transcripts have been inferred from <a href="http://www.ebi.ac.uk/%7Eguy/exonerate/">Exonerate</a> alignments of wheat coding sequences (CDS) from two sets of transcripts: <em>Triticum turgidum</em> assembled RNAseq data (<a href="http://europepmc.org/search?page=1&query=AUTH:%22Krasileva+KV%22">Krasileva <em>et al.</em></a>, Genome Biology 2013, 14:R66, Supplemental dataset 7) and a collection of publicly available wheat transcripts filtered to exclude pseudogenes, sequences shorter than 90 bp, and ORFs similar to those present in the <em>T. turgidum</em> set. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?r=5D:152946887-152957574;contigviewbottom=transcript_core_tgac_prediction_supp7,transcript_core_tgac_prediction_supp17=normal">Click here for example</a>. The program findorf was used to predict the CDS within these transcripts as described in Krasileva <em>et al.</em> [4]. See <a href="http://maswheat.ucdavis.edu/Transcriptome/">Triticeae-CAP</a> project page for more information.</p>

<h4>Repeat feature and non-coding RNA annotation</h4>

<p><a href="http://www.girinst.org/repbase/" title="Repbase">Repbase</a> repeats as well as Triticeae repeats from <a href="http://wheat.pw.usda.gov/ITMI/Repeats/"><em>TREP</em></a> were aligned to the <em>T. aestivum </em>genome using RepeatMasker as part of our <a href="http://ensemblgenomes.org/info/data/repeat_features">standard repeat feature annotation pipeline</a>.</p>

<p>Non-coding RNA genes have been annotated using tRNAScan-SE (Lowe, T.M. and Eddy, S.R. 1997), RFAM (Griffiths-Jones et al 2005), and RNAmmer (Lagesen K.,et al 2007) as part of our <a href="http://ensemblgenomes.org/info/data/ncrna">standard n</a><a href="http://ensemblgenomes.org/info/data/ncrna">on-coding RNA annotation </a><a href="http://ensemblgenomes.org/info/data/ncrna">pipeline</a>.</p>

<!-- {annotation} -->
<!-- {regulation} --><a name="regulation"></a>
<h3 id="regulation">Sequence alignments</h3>

<h4>Transcriptome mappings</h4>

<p>Wheat RNA-Seq, ESTs, and UniGene datasets have been aligned to the <em>Triticum aestivum</em> genome:</p>

<ul><li>454 RNA-seq data were aligned using <a href="http://code.google.com/p/rna-star/">STAR</a>, for the following ENA studies:

	<ul><li><a href="http://www.ebi.ac.uk/ena/data/view/SRP002455">SRP002455</a>, submitted by Akhunova et al [5]. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=SRS072840=tiling">Click here for example</a>.</li>
		<li><a href="http://www.ebi.ac.uk/ena/data/view/ERP001415">ERP001415</a>, submitted by Brenchley et al [6]. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=ERS137641=tiling">Click here for example</a>.</li>
		<li><a href="http://www.ebi.ac.uk/ena/data/view/SRP004502">SRP004502</a>.<!--nbsp--> <!--nbsp--><a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=SRS139097=tiling">Click here for example.</a></li>
	</ul></li>
	<li>Illumina RNA-seq data were aligned using <a href="http://code.google.com/p/rna-star/">STAR</a>, for the following ENA study:
	<ul><li><a href="http://www.ebi.ac.uk/ena/data/view/ERP004505">ERP004505</a>, submitted by Pfeifer et al. [7]. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=ERS386441=normal">Click here for example</a>.</li>
	</ul></li>
	<li>Wheat UniGene cluster sequence data were aligned using <a href="http://www.ebi.ac.uk/%7Eguy/exonerate/">Exonerate</a>, following the standard Ensembl pipeline. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=dna_align_otherfeatures_wheat_unigene_exonerate=normal">Click here for example.</a></li>
	<li>All publicly available wheat EST data were aligned using <a href="https://code.google.com/p/rna-star/">STAR</a>. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=5D:152946887-152957574;contigviewbottom=dna_align_otherfeatures_wheat_est_star=normal">Click here for example</a>.</li>
	<li><a href="http://trifldb.psc.riken.jp/v3/index.pl">TriFLDB</a> [8] sequences were aligned using <a href="https://code.google.com/p/rna-star/">STAR</a>. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=1A:33304741-33307940;contigviewbottom=dna_align_otherfeatures_trifldb_est_star=labels">Click here for example</a>.</li>
</ul><h4>Analysis of the bread wheat genome using comparative whole genome shotgun sequencing - Brenchley <em>et al.</em> [6]</h4>

<p>The wheat genome assemblies previously generated by Brenchley <em>et al. </em>(PMID:<a href="http://europepmc.org/abstract/MED/23192148">23192148</a>) have also been aligned to the survey sequence, <em>Brachypodium</em>, barley and the wild wheat progenitors (<em>Triticum urartu</em> and <em>Aegilops tauschii</em>). Homoeologous variants inferred between the three wheat genomes (A, B, and D) are displayed in the context of the gene models of these five genomes.</p>

<p>Sequences of diploid progenitor and ancestral species permitted homoeologous variants to be classified into two groups, 1) SNPs that differ between the A and D genomes (where the B genome is unknown) and, 2) SNPs that are the same between the A and D genomes, but differ in B.</p>

<p>The wheat gene alignments and the projected wheat SNPs are available on the Location view of the <em>Triticum aestivum</em>, <em>Brachypodium distachyon</em> and <em>Hordeum vulgare</em> genomes, as additional tracks under the "Wheat SNPs and alignments" section of the "Configure This page" menu.<!--nbsp--> <!--nbsp--><a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;r=1A:33304741-33307940;contigviewbottom=Lastz_Wheat_A_subsequences_alignments=normal,Lastz_Wheat_B_subsequences_alignments=normal,Lastz_Wheat_D_subsequences_alignments=normal,Lastz_Wheat_X_subsequences_alignments=normal,simple_core_wheat_snp_a=normal,simple_core_wheat_snp_b=normal">Click here for a bread wheat example</a>.<!--nbsp--> <!--nbsp--><a href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=5:2580560-2583709;contigviewbottom=Lastz_Wheat_A_subsequences_alignments=normal,Lastz_Wheat_B_subsequences_alignments=normal,Lastz_Wheat_D_subsequences_alignments=normal,Lastz_Wheat_X_subsequences_alignments=normal,simple_core_wheat_snp_a=normal,simple_core_wheat_snp_b=normal" title="Example wheat gene alignments and homoeologous SNPs">Click here for a <em>Brachypodium</em> example</a>. <a href="http://plants.ensembl.org/Hordeum_vulgare/Location/View?db=core;r=1:279279843-279283843;contigviewbottom=Lastz_Wheat_A_subsequences_alignments=normal,Lastz_Wheat_B_subsequences_alignments=normal,Lastz_Wheat_D_subsequences_alignments=normal,Lastz_Wheat_X_subsequences_alignments=normal,simple_core_wheat_snp_a=normal,simple_core_wheat_snp_b=normal">Click here for a barley example</a>.</p>

<h4>Transcriptome assembly in diploid einkorn wheat <em>Triticum monococcum</em> - Fox et al. [9]</h4>

<p>Genome-wide transcriptomes of two <em>Triticum monococcum</em> subspecies were constructed, the wild winter wheat <em>T. monococcum ssp. aegilopoides</em> (accession G3116) and the domesticated spring wheat <em>T. monococcum ssp. monococcum</em> (accession DV92) by generating <em>de novo</em> assemblies of RNA-Seq data derived from both etiolated and green seedlings. Assembled data is available from the <a href="http://jaiswallab.cgrb.oregonstate.edu/genomics/wheat">Jaiswal lab</a> and raw reads are available from INSDC projects <a href="http://www.ebi.ac.uk/ena/data/view/PRJNA203221">PRJNA203221</a> and <a href="http://www.ebi.ac.uk/ena/data/view/PRJNA195398">PRJNA195398</a>.</p>

<p>The de novo transcriptome assemblies of DV92 and G3116 represent 120,911 and 117,969 transcripts, respectively. They were mapped to the bread wheat, barley and <em>Triticum urartu</em> genomes using <a href="https://code.google.com/p/rna-star/">STAR</a>. <a href="http://plants.ensembl.org/Triticum_aestivum/Location/View?db=core;g=Traes_4AL_E4470B79C;r=4A:184660209-184666527;t=Traes_4AL_E4470B79C.2;contigviewbottom=dna_align_otherfeatures_tmdv92_cdna_star=labels">Click here for a bread wheat example</a>.</p>

<!-- {regulation} -->
<!-- {variation} --><a name="variation"></a>
<h3 id="variation">Variation</h3>

<p>~504,092 SNP markers provided by CerealsDB, from the University of Bristol, were mapped to the TGACv1 assembly, running on ungapped model, with the following filtering criteria, 100% coverage, and 100% identity match.</p>

<p>These SNPs can be part of the following platforms:</p>

<ul><li>The Axiom 820K SNP Array (contains ~820,000 SNP markers of which 504,092 have been mapped).</li>
	<li>The Axiom 35K SNP Array (contains 35,000 SNP markers of which 21,423 have been mapped).</li>
</ul><p>Note that a SNP marker can be part of more than one platform.</p>

<!-- {variation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Wheat sequence search v2.0 online</h3>

<p>Full sequence-based searching of the wheat genome is now available within the standard Ensembl Genomes sequence search facilities (<a href="http://plants.ensembl.org/Multi/enasearch">ENA search</a> and <a href="http://plants.ensembl.org/Triticum_aestivum/Tools/Blast?db=core">BLAST</a>). The previous custom wheat-only search has now been discontinued.</p>

<h2>Links</h2>

<ul><li><a href="http://www.wheatgenome.org/">International Wheat Genome Sequencing Consortium (IWGSC)</a></li>
	<li><a href="http://wheat-urgi.versailles.inra.fr/">URGI Wheat Portal</a></li>
	<li><a href="http://www6.clermont.inra.fr/umr1095_eng/">GDEC Portal</a></li>
	<li><a href="http://pgsb.helmholtz-muenchen.de/plant/wheat//iwgsc/index.jsp">PGSB International Wheat Survey Genome Database</a></li>
	<li><a href="http://pgsb.helmholtz-muenchen.de/plant/wheat/uk454survey/index.jsp">PGSB 5x 454 Survey Wheat Genome Database</a></li>
	<li><a href="http://www.wheatgenome.org.uk/">Triticeae Genomics For Sustainable Agriculture resource page</a></li>
	<li><a href="http://maswheat.ucdavis.edu/Transcriptome/">Triticeae-CAP</a> in UC Davis University</li>
	<li><a href="http://jaiswallab.cgrb.oregonstate.edu/genomics/wheat"><em>Triticum monococcum</em> resources</a> from Jaiswal Lab in Oregon State University</li>
	<li><a href="http://wheat.pw.usda.gov/ITMI/Repeats/"><em>TREP</em></a>, the Triticeae Repeat Sequence Database</li>
	<li><a href="http://trifldb.psc.riken.jp/v3/index.pl">TriFLDB</a>, the triticeae full-length CDS database</li>
	<li>ENA study <a href="http://www.ebi.ac.uk/ena/data/view/ERP000319">ERP000319</a>: 454 pyrosequencing of the <em>Triticum aestivum</em> (bread wheat) genome to 5X coverage</li>
	<li>ENA study <a href="http://www.ebi.ac.uk/ena/data/view/ERP001415">ERP001415</a>: 454 sequencing of <em>Triticum aestivum</em> (bread wheat) cv. Chinese spring cDNA samples from a pool of tissues, from plants under drought stress and from circadian-sampled leaves</li>
	<li>ENA study <a href="http://www.ebi.ac.uk/ena/data/view/ERP004505">ERP004505</a>: Analysis of the bread wheat grain transcriptome reveals complex genome interplay in a hexaploid cereal</li>
	<li><em>Triticum aestivum</em> ESTs at <a href="http://www.ebi.ac.uk/ena/data/warehouse/search?query=%22dataclass=%22EST%22%20AND%20tax_eq%284565%29%22&domain=sequence">ENA</a></li>
	<li><em>Triticum aestivum</em> Unigene cluster sequences at <a href="http://www.ncbi.nlm.nih.gov/unigene/?term=txid4565[Organism%3Anoexp]">NCBI</a></li>
	<li><a href="http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/indexNEW.php">CerealsDB</a> from the Functional Genomics Group at the University of Bristol</li>
	<li><a href="http://wheatgenomics.plantpath.ksu.edu/hapmap/">Wheat Hapmap project</a></li>
</ul>
<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/25035500">A chromosome-based draft sequence of the hexaploid bread wheat (Triticum aestivum) genome</a>.<br>2014. Science. 345:1251788.</li>
<li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/25637298">A whole-genome shotgun approach for assembling and anchoring the hexaploid bread wheat genome</a>.<br>Chapman JA, Mascher M, Bulu A, Barry K, Georganas E, Session A, Strnadova V, Jenkins J, Sehgal S, Oliker L et al. 2015. Genome Biol. 16:26.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/25035497">Structural and functional partitioning of bread wheat chromosome 3B</a>.<br>Choulet F, Alberti A, Theil S, Glover N, Barbe V, Daron J, Pingault L, Sourdille P, Couloux A, Paux E et al. 2014. Science. 345:1249721.</li>
<li><a id="ref-04"></a><a href="http://europepmc.org/abstract/MED/23800085">Separating homeologs by phasing in the tetraploid wheat transcriptome</a>.<br>Krasileva KV, Buffalo V, Bailey P, Pearce S, Ayling S, Tabbita F, Soria M, Wang S, Consortium I, Akhunov E et al. 2013. Genome Biol. 14:R66.</li>
<li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/20849627">Homoeolog-specific transcriptional bias in allopolyploid wheat</a>.<br>Akhunova AR, Matniyazov RT, Liang H, Akhunov ED. 2010. BMC Genomics. 11:505.</li>
<li><a id="ref-06"></a><a href="http://europepmc.org/abstract/MED/23192148">Analysis of the bread wheat genome using whole-genome shotgun sequencing</a>.<br>Brenchley R, Spannagl M, Pfeifer M, Barker GL, D'Amore R, Allen AM, McKenzie N, Kramer M, Kerhornou A, Bolser D et al. 2012. Nature. 491:705-710.</li>
<li><a id="ref-07"></a><a href="http://europepmc.org/abstract/MED/25035498">Genome interplay in the grain transcriptome of hexaploid bread wheat</a>.<br>Pfeifer M, Kugler KG, Sandve SR, Zhan B, Rudi H, Hvidsten TR, , Mayer KF, Olsen OA. 2014. Science. 345:1250091.</li>
<li><a id="ref-08"></a><a href="http://europepmc.org/abstract/MED/19448038">TriFLDB: a database of clustered full-length coding sequences from Triticeae with applications to comparative grass genomics</a>.<br>Mochida K, Yoshida T, Sakurai T, Ogihara Y, Shinozaki K. 2009. Plant Physiol. 150:1135-1146.</li>
<li><a id="ref-09"></a><a href="http://europepmc.org/abstract/MED/24821410">De Novo Transcriptome Assembly and Analyses of Gene Expression during Photomorphogenesis in Diploid Wheat Triticum monococcum</a>.<br>Fox SE, Geniza M, Hanumappa M, Naithani S, Sullivan C, Preece J, Tiwari VK, Elser J, Leonard JM, Sage A et al. 2014. PLoS ONE. 9:e96855.</li>
<li><a id="ref-10"></a><a href="http://europepmc.org/abstract/MED/22943283">CerealsDB 2.0: an integrated resource for plant breeders and scientists</a>.<br>Wilkinson PA, Winfield MO, Barker GL, Allen AM, Burridge A, Coghill JA, Edwards KJ. 2012. BMC Bioinformatics. 13:219.</li>
<li><a id="ref-11"></a><a href="http://europepmc.org/abstract/MED/24646323">Characterization of polyploid wheat genomic diversity using a high-density 90000 single nucleotide polymorphism array</a>.<br>Wang S, Wong D, Forrest K, Allen A, Chao S, Huang BE, Maccaferri M, Salvi S, Milner SG, Cattivelli L et al. 2014. Plant Biotechnol. J.. </li>
<li><a id="ref-12"></a><a href="http://europepmc.org/abstract/MED/21627760">Transcript-specific, single-nucleotide polymorphism discovery and linkage analysis in hexaploid bread wheat (Triticum aestivum L.)</a>.<br>Allen AM, Barker GL, Berry ST, Coghill JA, Gwilliam R, Kirby S, Robinson P, Brenchley RC, D'Amore R, McKenzie N et al. 2011. Plant Biotechnol. J. 9:1086-1099.</li>
<li><a id="ref-13"></a><a href="http://europepmc.org/abstract/MED/25886949">A haplotype map of allohexaploid wheat reveals distinct patterns of selection on homoeologous genomes</a>.<br>Jordan KW, Wang S, Lun Y, Gardiner LJ, MacLachlan R, Hucl P, Wiebe K, Wong D, Forrest KL,  et al. 2015. Genome Biol. 16:48.</li></ol>