<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p>Project funding: National Natural Science Foundation of China (grants # 30770143, 30621001 and 31171231) and the State Key Laboratory of Plant Genomics of China (grants # 2009B0714-02 and 2010B0527-01).<!--nbsp--> <!--nbsp--> These pre-publication data are being released under guidelines of the Fort Lauderdale Agreement, which reaffirms the balance between fair use (i.e. no pre-emptive publication) and early disclosure. Users are encouraged use these data to advance their research on individual loci but are asked to respect the rights of the investigators who generated these data to publish the whole-genome level description of the <em>O. brachyantha</em> in a peer-reviewed journal. This description includes whole-genome comparative analyses, genome size evolution, gene family evolution, gene organization and movement, heterochromatin, centromere evolution. This genome falls under the scope of the I-OMAP (International Oryza Map Alignment Project) consortium. The I-OMAP consortium is an internationally coordinated effort to create high-quality reference assemblies representing the diversity of wild and crop-progenitor species in the genus Oryza (<a href="http://dx.doi.org/10.1016/j.pbi.2013.02.014">Jacquemin et al, 2012</a>). For inquiries and information on how to cite these data please contact <a href="mailto:jfchen@genetics.ac.cn">Dr. Mingsheng Chen</a> or <a href="mailto:rwing@ag.arizona.edu">Dr. Rod Wing</a>.</p></div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Oryza brachyantha</em></h2>
<p><em>Oryza brachyantha</em> (wild rice) is a distant relative of cultivated rice (<em><a href="/Oryza_sativa" title="Oryza sativa in Ensembl Plants">O. satia Japonica</a></em> and <em><a href="/Oryza_indica" title="Oryza indica in Ensembl Plants">O. sativa Indica</a></em>). It is placed on the basal lineage of <em>Oryza</em> and is the only member of the genus assigned to the F-genome type. An annual or weekly perennial tufted grass, it is distributed in west and central Africa, and grows in open wetland habitats. It has potentially useful traits for rice breeding, including resistance/tolerance to <a href="http://www.knowledgebank.irri.org/ipm/stem-borer/stem-borer-species/yellow-stem-borer.html">yellow stem borer</a>, <a href="http://www.knowledgebank.irri.org/RiceDoctor/information-sheets-mainmenu-2730/pests-mainmenu-2737/rice-leaffolder-mainmenu-2816.html">leaffolder</a>, <a href="http://www.knowledgebank.irri.org/RiceDoctor/information-sheets-mainmenu-2730/pests-mainmenu-2737/rice-whorl-maggot-mainmenu-2819.html">whorl maggot</a> and <a href="http://www.knowledgebank.irri.org/RiceDoctor/information-sheets-mainmenu-2730/diseases-mainmenu-2735.html">bacterial blight</a>. The <em>O. brachyantha</em> genome provides an important resource for functional and evolutionary studies in the genus Oryza. Its nuclear genome is diploid (2n = 24) and ~362 Mbp.</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>
<p>The genome of <em>Oryza brachyantha</em> was sequenced and assembled in the laboratory of <a href="http://sourcedb.cas.cn/sourcedb_genetics_cas/yw/zjrc/pgr/200907/t20090721_2130990.html">Dr. Mingsheng Chen</a> at the <a href="http://english.genetics.cas.cn/">Institute of Genetics and Developmental Biology (IGDB)</a>, <a href="http://english.cas.cn/">Chinese Academy of Sciences</a>, Beijing, China [1]. A whole genome shotgun sequencing approach was used to generate ~ 104X coverage using the Illumina GA II platform. The genome was initially assembled using <a href="http://seqanswers.com/wiki/SOAPdenovo">SOAPdenovo</a> at <a href="http://www.genomics.cn/en/index">BGI-Shenzhen</a>, followed by integration with BAC-end sequences generated at the <a href="http://www.genome.arizona.edu/">Arizona Genomics Institute (AGI)</a>. The final assembled sequence was 261 Mbp with a scaffold N50 size of 1.6 Mbp.</p>
<p>Ordering of the scaffolds into twelve pseudomolecules was accomplished by anchoring to the BAC-based physical map generated at AGI, and confirmed by a cytogenetic approaches. The current assembly is v1.4 and is deposited in DDBJ/EMBL/GenBank under the accession <a href="http://www.ncbi.nlm.nih.gov/nuccore/AGAT00000000.1">AGAT00000000</a>.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>
<p>Protein-coding genes were annotated using an evidence-based approach as implemented in the <a href="http://www.gramene.org/">Gramene</a> gene-build pipeline by collaboration with Dr. Chengzhi Liang at IGDB. Transposable elements were annotated by collaboration with <a href="http://www.plantbreeding.uga.edu/people/jacksonPBGGatCAESUGA.html">Dr. Scott A. Jackson at University of Georgia</a><!--nbsp--> <!--nbsp-->[1].<!--nbsp--> <!--nbsp--><span style="font-size: 12.7272720336914px; line-height: 19.0909080505371px;">Non coding RNA genes were predicted with Infernal and tRNA genes with tRNAscan<!--nbsp--> <!--nbsp--></span><span style="font-size: 12.7272720336914px; line-height: 19.0909080505371px;">at Arizona Genomics Institute (AGI) led by Dr. Rod Wing</span><span style="font-size: 12.7272720336914px; line-height: 19.0909080505371px;">.</span><span style="font-size: 12.7272720336914px; line-height: 19.0909080505371px;"><!--nbsp--> <!--nbsp--></span></p>
<!-- {annotation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Gramene/Ensembl Genomes Annotation</h3>
<p style="color: #555555; font-size: 12.727272033691406px; font-weight: normal;">Additional annotations generated by the Gramene project include:</p>
<ul style="color: #555555; font-size: 12.727272033691406px; font-weight: normal;"><li>Gene phylogenetic trees with other<!--nbsp--> <!--nbsp-->Gramene<!--nbsp--> <!--nbsp-->species,<!--nbsp--> <!--nbsp--><a href="/Oryza_brachyantha/Gene/Compara_Tree?g=OB11G27430;r=11:15547479-15547857;t=OB11G27430.1">see example</a>.</li>
<li>Lastz Whole Genome Alignment to<!--nbsp--> <!--nbsp--><i>Oryza sativa</i><!--nbsp--> <!--nbsp-->Japonica (IRGSP v1),<!--nbsp--> <!--nbsp--><em>Arabidopsis thaliana</em> and a few other AA genomes,<!--nbsp--> <!--nbsp--><a href="/Oryza_brachyantha/Location/Compara_Alignments/Image?align=9169&db=core&r=1%3A8001-18000">see example</a>.</li>
<li>Ortholog based<!--nbsp--> <!--nbsp--><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247098">DAGchainer</a><!--nbsp--> <!--nbsp-->synteny detection against other AA genomes,<!--nbsp--> <!--nbsp--><a href="/Oryza_brachyantha/Location/Synteny?r=1:8001-18000">see example</a>.</li>
<li>Mapping to the genome of multiple sequence-based feature sets using<!--nbsp--> <!--nbsp--><a href="http://docs.gramene.org/Blat_pipeline:_sequence_mapping_analysis">gramene blat pipeline</a>,<!--nbsp--> <!--nbsp--><a href="/Oryza_brachyantha/Location/View?r=1:8001-18000">see example</a>.</li>
<li>Identification of various repeat features from MIPS and AGI repeat libraries.</li>
</ul><h3></h3>
<h3 id="other">Links</h3>
<ul><li><a href="http://www.knowledgebank.irri.org/extension/oryza-brachyantha-a-chev-et-roehr.html"><em>O. brachyantha</em> at Rice Knowledge Bank</a></li>
</ul>
<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/23481403">Whole-genome sequencing of Oryza brachyantha reveals mechanisms underlying Oryza genome evolution</a>.<br>Chen J, Huang Q, Gao D, Wang J, Lang Y, Liu T, Li B, Bai Z, Luis Goicoechea J, Liang C et al. 2013. Nat Commun. 4:1595.</li></ol>