<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p>This assembly is available for use under the terms of the <a href="http://www.nature.com/nature/journal/v461/n7261/full/461168a.html">Toronto Agreement</a>.</p>

<p>This pre-site offers immediate public access to a new assembly of the wheat genome, which has annotation transferred from previous assemblies. For access to the main Ensembl Plants site, please click <a href="http://plants.ensembl.org/triticum_aestivum">here</a>.</p>
</div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<p>This Ensembl Plants pre-site contains the first release of the TGACv1 genome assembly of <em>Triticum aestivum</em> cv. Chinese Spring, generated by <a href="http://www.tgac.ac.uk/">The Genome Analysis Centre, Norwich</a>, as part of the BBSRC-funded project, <a href="http://www.wheatgenome.org.uk">Triticeae Genomics for Sustainable Agriculture.</a> The assembly has an N50 of 88 Kb and a total length of 13.4 Gb in contigs greater than 500 bp.</p>

<p>A total of 98,974 genes (99% of the total) annotated on the previously released assembly have been located on the new assembly. Alignments of RNA-seq data from 3 different studies across 18 samples have additionally been located on the new assembly. Data can be searched via gene identifier in the text search box, or via BLAST. Further annotation will be progressively available over the coming months.</p>

<p>The main Ensembl Plants site (<a href="http://plants.ensembl.org/Triticum_aestivum/Info/Index">http://plants.ensembl.org/Triticum_aestivum/Info/Index</a>) continues to offer a more mature set of annotation on an assembly comprising the IWGSC Chromosome Survey Sequence data (<a href="http://europepmc.org/abstract/MED/25035500">IWGSC et al. 2015</a>) supplemented by the chromosome 3B assembly (<a href="http://europepmc.org/abstract/MED/25035497">Choulet et al. 2015</a>).</p>

<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3>Assembly</h3>

<p>The first release of the TGACv1 genome assembly of <em>Triticum aestivum</em> cv. Chinese Spring, generated by <a href="http://www.tgac.ac.uk/">The Genome Analysis Centre, Norwich</a>, as part of the BBSRC-funded project, <a href="http://www.wheatgenome.org.uk">Triticeae Genomics for Sustainable Agriculture.</a> The assembly has an N50 of 88 Kb and a total length of 13.4 Gb in contigs greater than 500 bp.</p>

<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>

<p>A total of 43,766 'full length' IWGSC genes annotated on the previously released IWGSC CSS assembly have been projected to the TGACv1 assembly using alignments generated by the Assembly To Assembly Comparator (ATAC). An additional 98,974 genes (99% of the total) have been located on the new assembly using exon by exon using BLASTN.</p>

<p>RNA-Seq data from 3 different studies across 18 samples, from different tissues and developmental stages, have been aligned.</p>

<p>Further annotation will be progressively available over the coming months.</p>

<!-- {annotation} -->
<!-- {variation} --><a name="variation"></a>
<h3>Variation</h3>

<p>A total of 476,541 variation (marker) features from the previous IWGSC assembly using alignments generated by the Assembly To Assembly Comparator (ATAC).</p>

<p>Variation (marker) features were aligned from CerealsDB using LASTZ.</p>

<p>Variation sets used where Axiom 820K and 35K SNP arays.</p>

<p>A total of 504,092 variation features were aligned and added to the DB</p>

<p>For more information about CerealsDB as see <a href="http://www.cerealsdb.uk.net/cerealgenomics/CerealsDB/indexNEW.php">here</a>.</p>

<!-- {variation} -->
