<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p>Project funding: National Science Foundation Plant Genome Research Program (#082224) awarded to R. Wing, S. Rounsley and Y. Yu. This genome falls under the scope of the I-OMAP (International Oryza Map Alignment Project) consortium. The I-OMAP consortium is an internationally coordinated effort to create high-quality reference assemblies representing the diversity of wild and crop-progenitor species in the genus Oryza (<a href="http://dx.doi.org/10.1016/j.pbi.2013.02.014">Jacquemin et al, 2012</a>).</p></div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Oryza glaberrima</em></h2>
<p><em>Oryza glaberrima</em> (African rice) is a cultivated grain distinct from its better known cousin <em><a title="Oryza indica in Ensembl Plants" href="/Oryza_indica">Oryza sativa</a></em> (Asian rice). African rice was independently domesticated ~3000 years ago in the Niger River Delta from its still extant progenitor, <em>Oryza barthii</em>. While lacking many of the agronomic and quality traits found in Asian rice, <em>O. glaberrima</em> is significant for its resistance to many pests and diseases and for its tolerance of drought and infertile soils. Interspecific crosses between African and Asian rice have produced cultivars with improved yield and quality traits, that have been adopted by many African countries to meet the growing need for rice as a staple food. From a scientific perspective the genome of <em>O. glaberrima</em> provides insight into the genetic basis of domestication and other traits by finding commonalities and differences with <em>O. sativa</em>. Similar to Asian rice, African rice is a diploid A-type genome, having 12 chromosomes and an estimated size of ~358 Mbp.</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>
<p>The genome sequence was generated and assembled by the Arizona Genomics Institute (AGI) using strain IRGC:96717. The current assembly is "Oryza_glaberrima_AGI1.1". It incorporates the previously assembled chromosome 3 short arm (Chr3s) sequence and consists of 12 chromosome pseudomolecules and 1,939 unplaced scaffolds. Chr3s was sequenced and assembled using a heavily manually edited physical map. BAC clones were shotgun Sanger sequenced to 8x coverage and phase II finished. Assembly of the tile sequence was performed manually. The rest of the genome was sequenced with a hybrid BAC pooling and whole genome shotgun approach with 30x coverage of Roche GSFLX 454 Titanium sequencing technology. Sequences were assembled and combined with a subset of previously sequenced BAC clones to produce a whole genome assembly. The underlying scaffolds have been deposited in GenBank with the accession number ADWL01000000.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>
<p>Protein-coding genes were annotated by the Munich Information Center for Protein Sequences (MIPS) led by Klaus Meyer using an evidence-based approach. Annotation of repeats and transposable elements was conducted at AGI. Prediction of ncRNA and tRNA genes was conducted at AGI.</p>
<!-- {annotation} -->
<!-- {variation} --><a name="variation"></a>
<h3 id="variation">Variation</h3>
<p>Variation data comes from two (unpublised) sources:</p>
<ol><li>20 diverse accessions of <em>Oryza glaberrima</em> and</li>
<li>19 accessions of its wild progenitor, <em>Oryza barthii</em>, collected from geographically distributed regions of Africa.</li>
</ol><p>Briefly, WGS reads were generated using low-coverage <a href="http://www.illumina.com">Illumina</a> sequencing. Filtered reads were aligned to O. glaberrima using <a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168">BWA</a> and SNP calling was done using a combination of <a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943">SAMtools</a> and <a href="http://www.broadinstitute.org/gatk">GATK</a> with standard quality and coverage filters giving a final set of ~8M SNPs.</p>
<p>These unpublished data were kindly contributed by Rod Wing of the <a href="http://www.genome.arizona.edu">Arizona Genomics Institute</a> and collaborator <a href="http://biology.umd.edu/faculty/carlosamachado">Carlos Machado</a> of the <a href="http://www.umd.edu">University of Maryland</a>, as part of the Oryza Genome Evolution project funded by <a href="http://www.nsf.gov/awardsearch/showAward?AWD_ID=1026200">NSF Award #1026200</a>.</p>
<!-- {variation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Gramene/Ensembl Genomes Annotation</h3>
<p>Additional annotations generated by the Gramene project include:</p>
<ul><li>Gene phylogenetic trees with other<!--nbsp--> <!--nbsp-->Gramene<!--nbsp--> <!--nbsp-->species,<!--nbsp--> <!--nbsp--><a href="/Oryza_glaberrima/Gene/Compara_Tree?g=ORGLA12G0072100;r=12:6645096-6649987;t=ORGLA12G0072100.1">see example</a>.</li>
<li>Lastz Whole Genome Alignment to<!--nbsp--> <!--nbsp--><em>Arabidopsis thaliana</em>,<!--nbsp--> <!--nbsp--><i>Oryza sativa</i><!--nbsp--> <!--nbsp-->Japonica (IRGSP v1),<!--nbsp--> <!--nbsp-->and a few AA oryza genomes,<!--nbsp--> <!--nbsp--><a href="/Oryza_glaberrima/Location/Compara_Alignments/Image?align=9160&db=core&r=1%3A8001-18000">see example</a>.</li>
<li>Ortholog based<!--nbsp--> <!--nbsp--><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247098">DAGchainer</a><!--nbsp--> <!--nbsp-->synteny detection against other AA genomes,<!--nbsp--> <!--nbsp--><a href="/Oryza_glaberrima/Location/Synteny?r=1:8001-18000">see example</a>.</li>
<li>Mapping to the genome of multiple sequence-based feature sets using<!--nbsp--> <!--nbsp--><a href="http://docs.gramene.org/Blat_pipeline:_sequence_mapping_analysis">gramene blat pipeline</a>,<!--nbsp--> <!--nbsp--><a href="/Oryza_glaberrima/Location/View?db=core;r=1:225001-275000">see example</a>.</li>
<li>Identification of various repeat features from MIPS and AGI repeat libraries.</li>
<li>Variation effect prediction with sequence ontology, <a href="/Oryza_glaberrima/Variation/Explore?r=12:18785206-18786206;v=tmp_12_18785706;vdb=variation;vf=4432574">for example</a>.</li>
</ul><h3 id="other">Links</h3>
<ul><li><a href="http://www2.genome.arizona.edu/genomes/rice_glab_genome_announce">Details on sequence and assembly</a></li>
<li><a href="http://www2.genome.arizona.edu/genomes/rice_glab_dwnload">Terms of Use and download</a></li>
<li><a href="http://ava.genome.arizona.edu:8081/GenomesWeb/home.seam"><em>O. glaberrima</em> repeats</a></li>
<li><a href="http://en.wikipedia.org/wiki/Oryza_glaberrima">General information on <em>Oryza glaberrima</em></a></li>
<li><a href="http://en.wikipedia.org/wiki/NERICA">New Rice for Africa</a></li>
</ul>
<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/21400109">Genetic diversity and domestication history of African rice (Oryza glaberrima) as inferred from multiple gene sequences</a>.<br>Li ZM, Zheng XM, Ge S. 2011. Theor. Appl. Genet.. 123:21-31.</li>
<li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/20626650">Rice structural variation: a comparative analysis of structural variation between rice and three of its closest relatives in the genus Oryza</a>.<br>Hurwitz BL, Kudrna D, Yu Y, Sebastian A, Zuccolo A, Jackson SA, Ware D, Wing RA, Stein L. 2010. Plant J.. 63:990-1003.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/21423767">Patterns of sequence divergence and evolution of the S orthologous regions between Asian and African cultivated rice species</a>.<br>Guyot R, Garavito A, Gavory F, Samain S, Tohme J, Ghesquire A, Lorieux M. 2011. PLoS ONE. 6:e17726.</li>
<li><a id="ref-04"></a><a href="http://europepmc.org/abstract/MED/21385395">Exceptional lability of a genomic complex in rice and its close relatives revealed by interspecific and intraspecific comparison and population analysis</a>.<br>Tian Z, Yu Y, Lin F, Yu Y, Sanmiguel PJ, Wing RA, McCouch SR, Ma J, Jackson SA. 2011. BMC Genomics. 12:142.</li>
<li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/21323774">Distinct evolutionary patterns of Oryza glaberrima deciphered by genome sequencing and comparative analysis</a>.<br>Sakai H, Ikawa H, Tanaka T, Numa H, Minami H, Fujisawa M, Shibata M, Kurita K, Kikuta A, Hamada M et al. 2011. Plant J.. 66:796-805.</li>
<li><a id="ref-06"></a><a href="http://europepmc.org/abstract/MED/20522726">Orthologous comparisons of the Hd1 region across genera reveal Hd1 gene lability within diploid Oryza species and disruptions to microsynteny in Sorghum</a>.<br>Sanyal A, Ammiraju JS, Lu F, Yu Y, Rambo T, Currie J, Kollura K, Kim HR, Chen J, Ma J et al. 2010. Mol. Biol. Evol.. 27:2487-2506.</li>
<li><a id="ref-07"></a><a href="http://europepmc.org/abstract/MED/20333229">Paleogenomic analysis of the short arm of chromosome 3 reveals the history of the African and Asian progenitors of cultivated rices</a>.<br>Roulin A, Chaparro C, Pigu B, Jackson S, Panaud O. 2010. Genome Biol Evol. 2:132-139.</li></ol>