<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p>The barley genome assembly in Ensembl Plants is version 082214v1. This is a gene-space assembly of <em>Hordeum vulgare </em>cv. <em>Morex</em> generated by the <a href="http://barleygenome.org">International Barley Genome Sequencing Consortium</a>, with contigs<!--nbsp--> <!--nbsp-->ordered into chromosomal pseudomolecules using<!--nbsp--> <!--nbsp-->POPSEQ data from <a href="http://europepmc.org/abstract/MED/24243933">Ariyadasa et al.</a> and<!--nbsp--> <!--nbsp--><a href="http://europepmc.org/abstract/MED/23998490">Mascher et al.</a><!--nbsp--> <!--nbsp-->The gene models are provided by the<!--nbsp--> <!--nbsp--><a href="http://pgsb.helmholtz-muenchen.de/plant/">PGSB</a><!--nbsp--> <!--nbsp-->group at the <a href="http://www.helmholtz-muenchen.de">German Research Centre for Environmental Health</a>. The data presented in Ensembl Plants has been organised as part of the U.K. <a href="http://www.barleygenome.org.uk">Barley Genome Sequencing Project</a>, funded by the <a href="http://www.bbsrc.ac.uk">Biotechnology and Biological Sciences Research Council</a>.</p>

<p><!--nbsp--> <!--nbsp--></p>

<p class="rtecenter"><a href="http://www.public.iastate.edu/~imagefpc/IBSC%20Webpage/IBSC%20Template-home.html"><img alt="IBSC logo" src="/img/barleypic1.jpg" style="height:53px; padding:5px; vertical-align:middle; width:96px" /></a> <a href="http://pgsb.helmholtz-muenchen.de/plant/index.jsp"><img alt="PGSB logo" src="/img/pgsb-logo.png" style="height:57px; vertical-align:middle; width:172px" /></a><!--nbsp--> <!--nbsp--><a href="http://www.bbsrc.ac.uk"><img alt="BBSRC logo" src="/img/bbsrc.png" style="height:51px; padding:5px; vertical-align:middle; width:136px" /></a> <a href="http://www.barleygenome.org.uk/"><img alt="UK Barley Genome Sequencing logo" src="/img/uk-barley-logo.png" style="height:54px; padding:5px; vertical-align:middle; width:117px" /></a><a href="http://www.ipk-gatersleben.de/en/"><img alt="IPK logo" src="/img/ipklogo.png" style="height:58px; padding:5px; vertical-align:middle; width:109px" /></a></p>
</div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Hordeum vulgare</em></h2>

<p><em>Hordeum vulgare</em> (barley) is the world's fourth most important cereal crop and an important model for ecological adaptation, having been cultivated in all temperate regions from the Arctic Circle to the tropics. It was one of the first domesticated cereal grains originating in the Fertile Crescent over 10,000 years ago. About two-thirds of the global barley crop is used for animal feed, while the remaining third underpins the malting, brewing, and distilling industries. Although the human diet is not a primary use, barley offers potential health benefits, and is still the major calorie source in several parts of the world. Barley is a diploid member of the grass family, making it a natural model for the genetics and genomics of the Triticeae tribe, including polyploid <a href="/triticum_aestivum" title="Triticum aestivum in Ensembl Plants">wheat</a> and rye. With a haploid genome size of ~5.3 Gbp in 7 chromosomes, it is one of the largest diploid genomes sequenced to date.</p>

<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>

<p>The barley genome assembly presented here was produced by the <a href="http://www.public.iastate.edu/~imagefpc/IBSC%20Webpage/IBSC%20Template-home.html">International Barley Genome Sequencing Consortium</a> (IBSC) [3], expanded and refined using <a href="ftp://ftp.ipk-gatersleben.de/barley-popseq/">population sequencing (POPSEQ) data generated by the IPK</a> [5,6]. The structure of the genome assembly differs significantly from conventional assemblies. In place of conventional scaffolding, sequence contigs have been anchored to the genome using multiple lines of evidence, including alignment to a FPC physical map, identification of genetic markers, and syntenic stratification. As the assembly has good coverage of the genic regions, it has been termed a "gene-ome", a fairly complete gene set, much of which is located and ordered. Full details of the IBSC assembly process can be found in [3], and details of the IPK POPSEQ anchoring process in [5] and [6].</p>

<p>In the IBSC assembly, ~2.6 million sequenced contigs were generated using whole-genome shotgun sequencing (WGS). ~723,000 of these are assigned to specific chromosomal positions. These are shown in the browser on chromosomes labeled 1-7. An additional 138,000 WGS contigs could be assigned to a specific chromosome arm, but not to a more specific location. These are shown in the browser on chromosomes labeled "_unordered" (e.g. 1H_unordered, 3HL_unordered, 3HS_unordered). "HL" and "HS" refer to chromosome long and short arms respectively. Note: the "_unordered" suffix was added in Ensembl Genomes release 26; in previous releases, these chromosomes were labeled with just an "H," e.g. "1H, 3HS, 5HL."</p>

<p>In the IPK POPSEQ assembly, 90 Morex x Barke individuals were sequenced at low depth, and the sequences aligned with the IBSC WGS contigs. A total of 5.1 million SNPs were called on this alignment. These SNPs were then integrated into a high-density SNP-based genetic map [8]. IBSC WGS contigs harboring these SNPs were then placed into this genetic framework.</p>

<p>Examination of common markers shows that the IBSC and POPSEQ anchoring strategies are highly congruent [5,6]. Thus, the assembly presented in Ensembl Genomes includes contigs anchored by both strategies, so as not to lose any sequence or gene assignments. In the current release, all WGS contigs anchored by POPSEQ are placed accordingly. Then, those WGS contigs not anchored by POPSEQ, but anchored in the IBSC assembly retain their former positions from the IBSC assembly.</p>

<p>In the Ensembl browser, sequence contigs (labeled morex_contig) are displayed at approximate base-pair coordinates inferred from the location of associated genetic markers in the barley genetic and physical maps.</p>

<p>The chloroplast genome component and its gene annotation are also present. This was imported from ENA entry, <a href="http://www.ebi.ac.uk/ena/data/view/JQ754651">KC912687</a>.</p>

<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>

<p>79,379 genes have been called by the IBSC [3]. 26,159 of these have homology support from at least one other reference genome and are designated "high-confidence". The remaining 53,220 genes are designated "low-confidence". These two classes are displayed with separate tracks in the Ensembl browser.</p>

<p>15,719 high-confidence genes could be directly associated with the barley genome scaffold (BGS). To associate additional genes, the genetically anchored and physically ordered gene backbone was compared to the reference genomes of<em> Oryza sativa</em>, <em>Sorghum bicolor</em>, and<em> Brachypodium distachyon</em>, and extended and highly-conserved synteny regions were exploited to determine a location and order additional genes in the overall framework. A total of 3,743 extra genes were placed on chromosomes by this approach.</p>

<!-- {annotation} -->
<!-- {regulation} --><a name="regulation"></a>
<h3 id="regulation">Regulation and sequence alignments</h3>

<h4>Regulation</h4>

<p>Mappings for probes from the <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-GEOD-33391/?array=A-AFFY-31">Barley1 GeneChip array</a>, the <a href="http://www.ebi.ac.uk/arrayexpress/arrays/A-GEOD-16130/">Agilent barley full-length cDNA array</a>, and the barley PGRC1 10k <a href="http://www.ebi.ac.uk/arrayexpress/arrays/A-IPKG-2/">A</a> and <a href="http://www.ebi.ac.uk/arrayexpress/arrays/A-IPKG-3/">B</a> array set can be viewed in the browser. For example, see the results for <a href="http://plants.ensembl.org/Hordeum_vulgare/Location/Genome?db=core;fdb=funcgen;ftype=ProbeFeature;g=MLOC_66074;id=Contig2083_s_at;ptype=pset;r=5:507102502-507103895">Contig2083_s_at</a>.</p>

<h4>Transcriptome assembly in diploid einkorn wheat <em>Triticum monococcum</em> - Fox et al. [7]</h4>

<p>Genome-wide transcriptomes of two <em>Triticum monococcum</em> subspecies were constructed, the wild winter wheat <em>T. monococcum ssp. aegilopoides</em> (accession G3116) and the domesticated spring wheat <em>T. monococcum ssp. monococcum</em> (accession DV92) by generating <em>de novo</em> assemblies of RNA-Seq data derived from both etiolated and green seedlings. Assembled data is available from the <a href="http://jaiswallab.cgrb.oregonstate.edu/genomics/wheat">Jaiswal lab</a> and raw reads are available from INSDC projects <a href="http://www.ebi.ac.uk/ena/data/view/PRJNA203221">PRJNA203221</a> and <a href="http://www.ebi.ac.uk/ena/data/view/PRJNA195398">PRJNA195398</a>.</p>

<p>The de novo transcriptome assemblies of DV92 and G3116 represent 120,911 and 117,969 transcripts, respectively. They were mapped to the bread wheat, barley and <em>Triticum urartu</em> genomes using <a href="https://code.google.com/p/rna-star/">STAR</a>. <a href="http://plants.ensembl.org/Hordeum_vulgare/Location/View?db=core;g=MLOC_59496;r=5HL:1252730-1262411;contigviewbottom=dna_align_otherfeatures_tmdv92_cdna_star=labels">Click here for a barley example</a>.</p>

<h4><em>Triticum aestivum</em> transcriptome</h4>

<p>Wheat RNA-Seq, EST and UniGene datasets have also been aligned to the <em>Hordeum vulgare</em> genome:</p>

<ul><li>454 RNA-seq data, from the study <a href="http://www.ebi.ac.uk/ena/data/view/ERP001415">ERP001415,</a> were aligned using <a href="http://www.molecularevolution.org/software/genomics/gmap">GMAP</a>. <a href="../../../Hordeum_vulgare/Location/View?db=core;r=1:279279843-279283843;contigviewbottom=Gmap_Wheat_ERR125556=normal,Gmap_Wheat_ERR125557=normal">Click here for example.</a></li>
	<li>All publicly available wheat EST data were aligned using <a href="http://www.ebi.ac.uk/%7Eguy/exonerate/">Exonerate</a>. <a href="http://plants.ensembl.org/Hordeum_vulgare/Location/View?db=core;r=1:279279843-279283843;contigviewbottom=dna_align_otherfeatures_wheat_est_exonerate=normal">Click here for example</a>.</li>
	<li>Wheat UniGene cluster sequence data were aligned using <a href="http://www.ebi.ac.uk/~guy/exonerate/">Exonerate</a>. <a href="../../../Hordeum_vulgare/Location/View?db=core;r=1:279279843-279283843;contigviewbottom=dna_align_otherfeatures_wheat_unigene_exonerate=normal">Click here for example.</a></li>
</ul>
<!-- {regulation} -->
<!-- {variation} --><a name="variation"></a>
<h3 id="variation">Variation</h3>

<h4>Barley variation data</h4>

<p>Five sources of barley variation data are shown:</p>

<ol><li>Variation data from WGS survey sequencing of four cultivars, Barke, Bowman, Igri, Haruna Nijo and a wild barley (H. spontaneum). The data was collected as described in [3].</li>
	<li>SNPs discovered from RNA-Seq performed on the embryo tissues of 9 spring barley varieties (Barke, Betzes, Bowman, Derkado, Intro, Optic, Quench, Sergeant and Tocada) and Morex using Illumina HiSeq 2000 [3].<!--nbsp--> <!--nbsp--></li>
	<li>~5 million variations from population sequencing of 90 Morex x Barke individuals [6]</li>
	<li>~6 million variations from population sequencing of 84 Oregon Wolfe barley individuals [6]</li>
	<li>SNPs from the <a href="http://bioinf.hutton.ac.uk/iselect/app/index.pl">Illumina iSelect 9k barley SNP chip</a>. ~2,600 markers associated with these SNPs [8] are also displayed.</li>
</ol><p>As of release 25, all barley variations have been added to the <a href="http://www.transplantdb.eu/variation/intro">transPLANT variation archive</a>.</p>

<h4><span style="color:#333333; font-size:12.8000001907349px">Wheat inter-homoeologous variants</span></h4>

<p>As part of the wheat genome analysis, we have aligned a set of<!--nbsp--> <!--nbsp--><a href="/triticum_aestivum" style="line-height: 1.5em;" title="Triticum aestivum in Ensembl Plants"><em>Triticum aestivum</em></a><!--nbsp--> <!--nbsp-->(bread wheat) homoeologous SNPs (SNPs between the component A, B, and D genomes of wheat) against the <em>Hordeum vulgare</em> genome.<!--nbsp--> <!--nbsp-->SNPs have been classified into two groups, 1) SNPs that differ between the A and D genomes (where the B genome is unknown) and, 2) SNPs that are the same between the A and D genomes, but differ in B [4].</p>

<p>The wheat sequence (<span style="color:rgb(85, 85, 85); font-family:luxi sans,helvetica,arial,geneva,sans-serif; font-size:12.8000001907349px">ENA study</span><span style="color:rgb(85, 85, 85); font-family:luxi sans,helvetica,arial,geneva,sans-serif; font-size:12.8000001907349px"><!--nbsp--> <!--nbsp--></span><a href="http://www.ebi.ac.uk/ena/data/view/ERP000319" rel="external" style="color: rgb(0, 0, 102); padding-right: 12px; font-family: 'Luxi Sans', Helvetica, Arial, Geneva, sans-serif; font-size: 12.8000001907349px; line-height: 16px; background-image: url(data:image/svg+xml,%3C%3Fxml%20version%3D%221.0%22%20encoding%3D%22UTF-8%22%3F%3E%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2210%22%20height%3D%2210%22%3E%0A%20%20%3Cg%20transform%3D%22translate%28-826.429%20-698.791%29%22%3E%0A%20%20%20%20%3Crect%20width%3D%225.982%22%20height%3D%225.982%22%20x%3D%22826.929%22%20y%3D%22702.309%22%20fill%3D%22%23ffffff%22%20stroke%3D%22%233366bb%22%2F%3E%0A%20%20%20%20%3Cg%3E%0A%20%20%20%20%20%20%3Cpath%20d%3D%22M831.194%20698.791h5.234v5.391l-1.571%201.545-1.31-1.31-2.725%202.725-2.689-2.689%202.808-2.808-1.311-1.311z%22%20fill%3D%22%233366bb%22%2F%3E%0A%20%20%20%20%20%20%3Cpath%20d%3D%22M835.424%20699.795l.022%204.885-1.817-1.817-2.881%202.881-1.228-1.228%202.881-2.881-1.851-1.851z%22%20fill%3D%22%23ffffff%22%2F%3E%0A%20%20%20%20%3C%2Fg%3E%0A%20%20%3C%2Fg%3E%0A%3C%2Fsvg%3E); background-position: 100% 50%; background-repeat: no-repeat;">ERP000319</a>)<!--nbsp--> <!--nbsp-->alignments<!--nbsp--> <!--nbsp-->and the projected homoeologous SNPs are available as tracks under the "Wheat SNPs and alignments" section of the "Configure This page" menu.<!--nbsp--> <!--nbsp--><a href="../../../Hordeum_vulgare/Location/View?db=core;r=1:279279843-279283843;contigviewbottom=Lastz_Wheat_A_subsequences_alignments=normal,Lastz_Wheat_B_subsequences_alignments=normal,Lastz_Wheat_D_subsequences_alignments=normal,Lastz_Wheat_X_subsequences_alignments=normal,simple_core_wheat_snp_a=normal,simple_core_wheat_snp_b=normal" style="line-height: 1.5em;" title="Example wheat gene alignments and homoeologous SNPs">Click here for example.</a></p>

<!-- {variation} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/19126706">The international barley sequencing consortium--at the threshold of efficient access to the barley genome</a>.<br>Schulte D, Close TJ, Graner A, Langridge P, Matsumoto T, Muehlbauer G, Sato K, Schulman AH, Waugh R, Wise RP et al. 2009. Plant Physiol.. 149:142-147.</li>
<li><a id="ref-02"></a><a href="http://europepmc.org/abstract/MED/6977">Nuclear dna amounts in angiosperms</a>.<br>Bennett MD, Smith JB. 1976. Philos. Trans. R. Soc. Lond., B, Biol. Sci.. 274:227-274.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/23075845">A physical, genetic and functional sequence assembly of the barley genome</a>.<br>International Barley Genome Sequencing Consortium, Mayer KF, Waugh R, Brown JW, Schulman A, Langridge P, Platzer M, Fincher GB, Muehlbauer GJ, Sato K et al. 2012. Nature. 491:711-716.</li>
<li><a id="ref-04"></a><a href="http://europepmc.org/abstract/MED/23192148">Analysis of the bread wheat genome using whole-genome shotgun sequencing</a>.<br>Brenchley R, Spannagl M, Pfeifer M, Barker GL, D'Amore R, Allen AM, McKenzie N, Kramer M, Kerhornou A, Bolser D et al. 2012. Nature. 491:705-710.</li>
<li><a id="ref-05"></a><a href="http://europepmc.org/abstract/MED/24243933">A sequence-ready physical map of barley anchored genetically by two million single-nucleotide polymorphisms</a>.<br>Ariyadasa R, Mascher M, Nussbaumer T, Schulte D, Frenkel Z, Poursarebani N, Zhou R, Steuernagel B, Gundlach H, Taudien S et al. 2014. Plant Physiol.. 164:412-423.</li>
<li><a id="ref-06"></a><a href="http://europepmc.org/abstract/MED/23998490">Anchoring and ordering NGS contig assemblies by population sequencing (POPSEQ)</a>.<br>Mascher M, Muehlbauer GJ, Rokhsar DS, Chapman J, Schmutz J, Barry K, Muoz-Amatrian M, Close TJ, Wise RP, Schulman AH et al. 2013. Plant J.. 76:718-727.</li>
<li><a id="ref-07"></a><a href="http://europepmc.org/abstract/MED/24821410">De Novo Transcriptome Assembly and Analyses of Gene Expression during Photomorphogenesis in Diploid Wheat Triticum monococcum</a>.<br>Fox SE, Geniza M, Hanumappa M, Naithani S, Sullivan C, Preece J, Tiwari VK, Elser J, Leonard JM, Sage A et al. 2014. PLoS ONE. 9:e96855.</li>
<li><a id="ref-08"></a><a href="http://europepmc.org/abstract/MED/23160098">Natural variation in a homolog of Antirrhinum CENTRORADIALIS contributed to spring growth habit and environmental adaptation in cultivated barley</a>.<br>Comadran J, Kilian B, Russell J, Ramsay L, Stein N, Ganal M, Shaw P, Bayer M, Thomas W, Marshall D et al. 2012. Nat. Genet.. 44:1388-1392.</li></ol><p>Picture credit: Lucash (Own work) [CC-BY-SA-3.0 (<a href="http://creativecommons.org/licenses/by-sa/3.0">http://creativecommons.org/licenses/by-sa/3.0</a>) or GFDL (<a href="http://www.gnu.org/copyleft/fdl.html">http://www.gnu.org/copyleft/fdl.html</a>)], via Wikimedia Commons</p>