<!-- {acknowledgement} --><a name="acknowledgement"></a>
<div class="info-box embedded-box"><p><span style="font-size: small;">These pre-publication data are being released under guidelines of <a href="http://www.genome.gov/10506537">the Fort Lauderdale Agreement</a>, which reaffirms the balance between fair use (i.e. no pre-emptive publication) and early disclosure. Users are encouraged use these data to advance their research on individual loci but are asked to respect the rights of the investigators who generated these data to publish the whole-genome level description of the <em>O. meridionalis</em> in a peer-reviewed journal. This description includes whole-genome comparative analyses, genome size evolution, gene family evolution, gene organization and movement, heterochromatin, centromere evolution. <span style="font-family: 'Luxi Sans',Helvetica,Arial,Geneva,sans-serif; line-height: 16px;">This genome falls under the scope of the I-OMAP (International Oryza Map Alignment Project) consortium. </span><span style="font-family: 'Luxi Sans',Helvetica,Arial,Geneva,sans-serif; line-height: 16px;"> </span><span style="font-family: 'Luxi Sans',Helvetica,Arial,Geneva,sans-serif; line-height: 16px;">The I-OMAP consortium is an internationally coordinated effort to create high-quality reference assemblies representing the diversity of wild and crop-progenitor species in the genus Oryza (<a href="http://dx.doi.org/10.1016/j.pbi.2013.02.014">Jacquemin et al, 2012</a>). </span>For inquiries and information on how to cite these data please contact <a href="mailto:rwing@ag.arizona.edu">Dr. Rod Wing</a>.</span></p></div>
<!-- {acknowledgement} -->
<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Oryza meridionalis</em></h2>
<p><em>Oryza meridionalis</em> is a wild rice found in Australia, one of the wild rice species included in the OMAP project. It belongs to the AA genome group. It has 12 chromosomes with a nuclear genome size of 435Mb (flow cytometry). It is found at edges of freshwater lagoons, temporary pools, and swamps in 15-20 cm of water and grows in black, clay soil in open habitats. This work was part of the Oryza Genome Evolution project funded by NSF Award #1026200 and in collaboration with Olivier Panaud (France) and Robert Henry (Australia).</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>
<p>The genome sequence was generated and assembled by the Arizona Genomics Institute (AGI) using accession <b>W2112</b>. Illumina reads of different library sizes were assembled with Abyss; scaffolds were constructed with SSPACE, and gaps were closed with GapFiller. The estimated coverage from the WGS was 166x. Total sequence length 335,668,232 bp; Number of contigs 62,778; Contig N50: 9,149bp.</p>
<p>The current assembly was updated from v1 to v1.3 (which mainly consisted in reducing gap size, breaking down or excluding contigs, and removing potentil contaminating sequences, e.g. contigs bearing similarity to mitochondrial sequences); MAKER-P annotations were also updated.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>
<p><span style="font-size: 12.7272720336914px; line-height: 19.2000007629395px;">Protein-coding gene annotation was performed with</span><span style="font-size: 12.7272720336914px; line-height: 19.2000007629395px;"><!--nbsp--> <!--nbsp--></span><span style="font-size: 12.7272720336914px; line-height: 19.2000007629395px;">evidence-based</span><span style="font-size: 12.7272720336914px; line-height: 19.2000007629395px;"><!--nbsp--> <!--nbsp-->MAKER-P genome annotation pipeline.</span><span style="font-size: 12.7272720336914px; line-height: 1.5em;"><!--nbsp--> <!--nbsp-->Non coding RNA genes were predicted with Infernal and tRNA genes with tRNAscan. RepeatMasker was used to annotate repeats and transposable elements with Oryza-specific de novo repeat libraries. These analyses<!--nbsp--> <!--nbsp--></span><span style="font-size: 12.7272720336914px; line-height: 19.0909080505371px;">were conducted at Arizona Genomics Institute (AGI) led by Dr. Rod Wing.</span></p>
<!-- {annotation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Gramene/Ensembl Genomes Annotation</h3>
<p>Additional annotations generated by the Gramene project include:</p>
<ul><li>Gene phylogenetic trees with other Gramene species: <a href="/Oryza_meridionalis/Gene/Compara_Tree?g=OMERI01G00010;r=1:10893-11879;t=OMERI01G00010.1">see example</a>.</li>
<li>Lastz Whole Genome Alignment to <i>Oryza sativa</i> Japonica (IRGSPv1), <a href="/Oryza_meridionalis/Location/Compara_Alignments/Image?align=9285&r=1%3A8001-18000">see example</a>.</li>
<li>Ortholog based <a href="http://www.ncbi.nlm.nih.gov/pubmed/15247098">DAGchainer</a> synteny detection against other AA genomes, <a href="/Oryza_meridionalis/Location/Synteny?r=1:8001-18000">see example</a>.</li>
<li>Mapping to the genome of multiple sequence-based feature sets using <a href="http://docs.gramene.org/Blat_pipeline:_sequence_mapping_analysis">gramene blat pipeline</a>, <a href="/Oryza_meridionalis/Location/View?r=Omeri_3s%3A8001-18000">see example</a>.</li>
<li>Identification of various repeat features from MIPS and AGI repeat libraries.</li>
</ul><h4>Links</h4>
<ul><li><a href="http://ensembl.gramene.org/species/oryza/rice_intro.html">Gramene species page for <i>Oryza</i> </a></li>
<li><a href="http://www.genome.arizona.edu/modules/publisher/item.php?itemid=7">Oryza Genome Evolution Project in AGI (Arizona Genomics Institute)</a></li>
<li><a href="http://rgp.dna.affrc.go.jp/E/IRGSP/index.html">International Rice Genome Sequencing Consortium (IRGSP)</a></li>
<li><a href="http://www.knowledgebank.irri.org/extension/oryza-meridionalis-ng.html">Rice Knowledge Bank</a></li>
<!--</div--></ul>
<!-- {other} -->
