<!-- {about} --><a name="about"></a>
<h2 id="about">About <em>Brachypodium distachyon</em></h2>
<p><em>Brachypodium distachyon</em>, like <em><a href="/Arabidopsis_thaliana" title="Arabidopsis thaliana in Ensembl Plants">Arabidopsis thaliana</a></em>, has several features that recommend it as a model plant for functional genomic studies, especially in the grasses. It has a small, diploid genome (~355 Mbp), small physical size, a short life-cycle and few growth requirements. Brachypodium is related to the major cereal grain species but is understood to be more closely related to the Triticeae (<a href="/Triticum_aestivum" title="Triticum aestivum in Ensembl Plants">wheat</a> and <a href="/Hordeum_vulgare" title="Hordeum vulgare in Ensembl Plants">barley</a>) than to the other cereals.</p>
<!-- {about} -->
<!-- {assembly} --><a name="assembly"></a>
<h3 id="assembly">Assembly</h3>
<p>Sequencing of <em>Brachypodium distachyon</em> inbred line Bd21 was performed by the US Department of Energy Joint Genome Institute (JGI) as part of its Community Sequencing Program [1]. Sequencing followed a whole genome shotgun strategy and achieved an average genome coverage of 8x. The assembly has been integrated with BAC-based physical maps and genetic linkage maps to produce a final sequence assembly of 83 scaffolds. Of these, 99.6% of the sequence was contained in five scaffolds, each representing one chromosome.</p>
<!-- {assembly} -->
<!-- {annotation} --><a name="annotation"></a>
<h3 id="genebuild">Annotation</h3>
<p>We have updated gene annotation (Brachy1.2). Genes were predicted by MIPS/JGI as follows; protein coding gene models were derived from weighted consensus predictions based on several types of evidence: <em>ab initio</em> gene finders, protein homology and optimal spliced alignments of expressed sequence tags (ESTs) and tentative consensus transcripts (TCs). A statistical combiner was trained and applied to the whole genome sequence to integrate experimental evidence into a consensus gene model for each locus. These gene models were rerun through the PASA pipeline to predict UTRs from EST information, to identify possible alternative splicing patterns, and to fit all predicted models to the splice sites supported by EST evidence. tRNA genes were identified by tRNA-SEscan using default parameters.</p>
<!-- {annotation} -->
<!-- {regulation} --><a name="regulation"></a>
<h3 id="regulation">Sequence alignments</h3>

<h4><em>Brachypodium sylvaticum</em> transcriptome</h4>
<p>De novo gene models from the RNA-Seq analysis of three <em>Brachypodium sylvaticum</em> populations [2] were mapped to the <em>B. distachyon</em> reference genome. <a href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=5:230439-232669;contigviewbottom=dna_align_core_brasy-cor=normal,dna_align_core_brasy-esp=normal,dna_align_core_brasy-gre,variation_feature_variation_Jaiswal_lab_OSU=normal">Click here for example</a>. Assembled data is available from the <a href="http://jaiswallab.cgrb.oregonstate.edu/genomics/brasy">Jaiswal lab</a> and raw reads are available from INSDC project <a href="http://www.ebi.ac.uk/ena/data/view/PRJNA182761">PRJNA182761</a>.</p>

<h4><em>Triticum aestivum</em> transcriptome</h4>
<p>Wheat RNA-Seq, EST and UniGene datasets have been aligned to the <em>Brachypodium distachyon</em> genome:</p>
<ul><li>454 Wheat RNA-seq data, from the study, <a href="http://www.ebi.ac.uk/ena/data/view/ERP001415">ERP001415</a>, were aligned using <a href="http://www.molecularevolution.org/software/genomics/gmap">GMAP</a>. <a href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=1:21899572-21905500;contigviewbottom=Gmap_Wheat_ERR125556=normal,Gmap_Wheat_ERR125557=normal">Click here for example.</a></li>
<li>All publicly available Wheat EST data were aligned using <a href="http://www.ebi.ac.uk/%7Eguy/exonerate/">Exonerate</a>, following the standard Ensembl pipeline. <a href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=1:21899572-21905500;contigviewbottom=dna_align_otherfeatures_wheat_est_exonerate=normal">Click here for example.</a></li>
<li>Wheat UniGene cluster sequence data were aligned using <a href="http://www.ebi.ac.uk/%7Eguy/exonerate/">Exonerate</a>, following the standard Ensembl pipeline. <a href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=1:21899572-21905500;contigviewbottom=dna_align_otherfeatures_wheat_unigene_exonerate=normal">Click here for example.</a></li>
</ul>
<!-- {regulation} -->
<!-- {variation} --><a name="variation"></a>
<h3 id="variation">Variation</h3>
<h4><em>Brachypodium</em> variation data</h4>
<p>Approximately 394,000 genetic variations have been identified by the alignment of transcriptome assemblies from three<!--nbsp--> <!--nbsp-->slender false brome (<em>Brachypodium sylvaticum</em>) populations [2]. Two<!--nbsp--> <!--nbsp-->populations come from<!--nbsp--> <!--nbsp--><em>B.<!--nbsp--> <!--nbsp-->sylvaticum</em>'s<!--nbsp--> <!--nbsp-->native range (Greece and Spain) and one comes from its invasive range (Oregon). Both the<!--nbsp--> <!--nbsp-->transciptome alignments and variation data are available in Ensembl Plants.<!--nbsp--> <!--nbsp--><a href="../../../Brachypodium_distachyon/Location/View?db=core;r=5:230439-232669;contigviewbottom=dna_align_core_brasy-cor=normal,dna_align_core_brasy-esp=normal,dna_align_core_brasy-gre,variation_feature_variation_Jaiswal_lab_OSU=normal">Click here for example</a>.</p>
<h4 id="wheat snp">Wheat inter-homoeologous variants</h4>
<p>As part of the wheat genome analysis, we have aligned a set of<!--nbsp--> <!--nbsp--><a href="/triticum_aestivum" title="Triticum aestivum in Ensembl Plants"><em>Triticum aestivum</em></a> (bread wheat) homoeologous SNPs (SNPs between the component A, B, and D genomes of wheat) against the <em>Brachypodium distachyon</em> genome.<!--nbsp--> <!--nbsp-->SNPs have been classified into two groups, 1) SNPs that differ between the A and D genomes (where the B genome is unknown) and, 2) SNPs that are the same between the A and D genomes, but differ in B [3].</p>
<p>The wheat sequence alignments and the projected homoeologous SNPs are available as tracks under the "Wheat SNPs and alignments" section of the "Configure This page" menu. <a title="Example wheat gene alignments and homoeologous SNPs" href="http://plants.ensembl.org/Brachypodium_distachyon/Location/View?db=core;r=5:2580560-2583709;contigviewbottom=Lastz_Wheat_A_subsequences_alignments=normal,Lastz_Wheat_B_subsequences_alignments=normal,Lastz_Wheat_D_subsequences_alignments=normal,Lastz_Wheat_X_subsequences_alignments=normal,simple_core_wheat_snp_a=normal,simple_core_wheat_snp_b=normal">Click here for example.</a></p>
<!-- {variation} -->
<!-- {other} --><a name="other"></a>
<h3 id="other">Links</h3>
<h4>Links (<em>Brachypodium distachyon</em>)</h4>
<ul><li><a href="http://www.gramene.org/Brachypodium_distachyon/Info/Index">Gramene species page for Brachypodium</a></li>
<li><a href="http://www.brachypodium.org">The International Brachypodium Initiative (IBI) community portal</a></li>
<li><a href="http://www.modelcrop.org">The John Innes Centre ModelCrop.org</a></li>
<li><a href="http://mips.helmholtz-muenchen.de/plant/brachypodium/">MIPS Brachypodium genome database</a></li>
<li><a href="http://www.phytozome.net/brachy.php">Phytozome entry page for <em>Brachypodium distachyon</em></a></li>
<li><a href="http://jaiswallab.cgrb.oregonstate.edu/genomics/brasy">Jaiswal Lab at Oregon State University</a></li>
<li><a href="http://www.ebi.ac.uk/ena/data/view/PRJNA182761">INSDC project PRJNA182761</a></li>
</ul><h4>Links (<em>Triticum aestivum</em>)</h4>
<ul><li><a href="http://mips.helmholtz-muenchen.de/plant/wheat/uk454survey/index.jsp">MIPS Wheat Genome Database</a></li>
<li>ENA study <a href="http://www.ebi.ac.uk/ena/data/view/ERP000319">ERP000319</a>: 454 pyrosequencing of the <em>Triticum aestivum</em> (bread wheat) genome to 5X coverage</li>
<li>ENA study <a href="http://www.ebi.ac.uk/ena/data/view/ERP001415">ERP001415</a>: 454 sequencing of <em>Triticum aestivum</em> (bread wheat) cv. Chinese spring cDNA samples from a pool of tissues, from plants under drought stress and from circadian-sampled leaves</li>
<li><em>Triticum aestivum</em> ESTs at <a href="http://www.ebi.ac.uk/ena/data/warehouse/search?query=%22dataclass=%22EST%22%20AND%20tax_eq%284565%29%22&domain=sequence">ENA</a></li>
<li><em>Triticum aestivum</em> UniGene cluster sequences at <a href="http://www.ncbi.nlm.nih.gov/unigene/?term=txid4565[Organism%3Anoexp]">NCBI</a></li>
</ul>
<!-- {other} -->
<h3>References</h3><ol><li><a id="ref-01"></a><a href="http://europepmc.org/abstract/MED/20148030">Genome sequencing and analysis of the model grass Brachypodium distachyon</a>.<br>The International Brachypodium Initiative. 2010. Nature. 463:763-768.</li>
<li><a id="ref-02"></a><a href="http://dx.doi.org/http://dx.doi.org/10.3732/apps.1200011">Sequencing and De Novo Transcriptome Assembly of Brachypodium sylvaticum (Poaceae)</a>.<br>Samuel E. Fox, Justin Preece, Jeffrey A. Kimbrel, Gina L. Marchini, Abigail Sage, Ken Youens-Clark, Mitchell B. Cruzan, and Pankaj Jaiswal. 2013. Applications in Plant Sciences. 1(3):1200011.</li>
<li><a id="ref-03"></a><a href="http://europepmc.org/abstract/MED/23192148">Analysis of the bread wheat genome using whole-genome shotgun sequencing</a>.<br>Brenchley R, Spannagl M, Pfeifer M, Barker GL, D'Amore R, Allen AM, McKenzie N, Kramer M, Kerhornou A, Bolser D et al. 2012. Nature. 491:705-710.</li></ol>